doc/user-guide/io.rst

```diff
@@ -828,8 +828,9 @@ For example:
 .. ipython:: python
 
     import zarr
+    from numcodecs.blosc import Blosc
```

Comment:
The [doc environment file](https://github.com/pydata/xarray/blob/main/ci/requirements/doc.yml) doesn't have numcodecs in it explicitly - should it?

Reply:
Numcodecs is a strict dependency of zarr so we can safely assume its there.